### PR TITLE
Bundle product: incorrect child item invoice_qty when dynamic price is set to false

### PR DESCRIPTION
**Pull request by engcom-Hotel**
### Description (*)
https://github.com/magento/magento2/issues/30802

### Fixed Issues (if relevant)
https://github.com/magento/magento2/issues/30802

### Manual testing scenarios (*)
https://github.com/magento/magento2/issues/30802

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests CE, Functional Tests EE

**Comment by engcom-Hotel** [comment]
Failing tests are not related to the changes.

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE

File: dev/tests/integration/testsuite/Magento/Sales/Controller/Adminhtml/Order/Invoice/SaveTest.php [hunk 1]
```diff
@@ -233,6 +233,40 @@ private function getOrderItemsQtyInvoiced(int $orderId): array
         return $connection->fetchCol($select);
     }
 
+    /**
+     * Get order items qty ordered
+     *
+     * @param int $orderId
+     * @return array
+     */
+    private function getOrderItemsQtyOrdered(int $orderId): array
```
Comment (eduard13) [inline dev/tests/integration/testsuite/Magento/Sales/Controller/Adminhtml/Order/Invoice/SaveTest.php]: Just as a note, `private` methods/properties should be declared after `public` declarations.

**Review by eduard13 — APPROVED** [review]
Good job 👍.
I've left a minor note.

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8705](https://jira.corp.magento.com/browse/ENGCOM-8705) has been created to process this Pull Request

File: dev/tests/integration/testsuite/Magento/Sales/Controller/Adminhtml/Order/Invoice/SaveTest.php (hunk 1 above)
Comment (engcom-Hotel) [reply dev/tests/integration/testsuite/Magento/Sales/Controller/Adminhtml/Order/Invoice/SaveTest.php]: Have been moved.

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Review by eduard13 — APPROVED** [review]
✅

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8705](https://jira.corp.magento.com/browse/ENGCOM-8705) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**

1. Create a new bundle product via Magento 2 administration
1.1. Change "Dynamic Pricing" setting to false
1.2. Add a price i.e. 75 $
1.3. Perhaps not relevant, but in my use case: [Add visible to "not visible individually"]
1.4. Add bundle options with setting "ship bundle items" set  to "together" <br>
1.4.1. Add one bundle option with a name of your choice of input type "checkbox" and set to "required"
1.4.2. Add two simple products of your choice (i.e. ID 1 and ID 2 of the demo content products) 
1.4.3. Change the qty settings of the simple products to a qty > 1 (i.e. 2 and 3)
1.4.4. leave the price range setting on "fixed"
1.4.5. Mark the checkbox "is Default"   
1.5. Save the product
2. Create a new order via Magento 2 administration 
2.1. Add the new bundle product to your order <br> 
2.2. Click on configure and only change the qty to 2 
2.3. Save the product items (uncheck the discount if it has been set, but dont recalculate)
2.4. Add a payment of your choice
2.5. Add a shipping of your choice <br>order
2.6. Save the order
3. Create a new invoice via Magento 2 administration
3.1. Skip the shipping procedure entirely, just create an invoice
3.2. Leave all settings as is
3.3. Save the invoice <br>invoiced
4. Check the database table `sales_order_items` selected on the order number (order id gets in `parent_id` column) <br>
(in our case the database values should be displayed in an pdf invoice, which also reflect the wrong qty's)

--> the order_qty has correctly been changed, but the invoice_qty has not been changed correctly and refers to the parent items qty... **see screenshots in the following sections**

 **Before:** :heavy_multiplication_x:  After creating the invoice the simple items within the bundle actually have the wrong qty - the qty of the parent item (the bundle item) instead. 

Screenshot from 2021-01-27 11-36-26

**After:** :heavy_check_mark:  After creating the invoice the simple items within the bundle have the correct qty

Screenshot from 2021-01-27 23-36-19

**Pull request merged**